portfolio_style.css

```diff
@@ -0,0 +1,155 @@
+@import url('https://fonts.googleapis.com/css?family=Charm|Fredericka+the+Great');
+
+html {
+	font-family: "Fredericka the Great";
+	color: #5f6161;
+	font-size: 10pt;
+	
+}
+
+*, *::before, *::after {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+	
+body {
+	background-color: white;
+}
+
+a {
+	color: white;
+	text-align: center;
+}
+
+a:hover {
+	color: #e00dbd
+}
+
+a:focus{
+	color:#e00dbd;
+}
+
+p {
+	text-align: center;
+}
+
+h1, h2 {
+	color: #b5b0b3;
+	text-align: center;
+}
+
+h1 {
+	font-size: 50pt;
+}
+
+.nav_bar {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	background-color: #e69ccb;
+	color: white;
+	height: 50px;
+	padding: 10px;
```

Comment:
use rem (5rem for these 2)

Reply:
sorry, I am confused... can we talk about this? ;)
I thought that rem was for font size only...

Reply:
Ok, I talked to Rayna and she said rem is used for fonts and paddings are OK in px. Cheers